src/__tests__/acceptance/ping.controller.acceptance.ts

```diff
@@ -0,0 +1,21 @@
+import {Client, expect} from '@loopback/testlab';
+import {FluffyOctoEngineApplication} from '../..';
+import {setupApplication} from './test-helper';
+
+describe('PingController', () => {
+  let app: FluffyOctoEngineApplication;
+  let client: Client;
+
+  before('setupApplication', async () => {
+    ({app, client} = await setupApplication());
+  });
+
+  after(async () => {
+    await app.stop();
+  });
+
+  it('invokes GET /ping', async () => {
+    const res = await client.get('/ping?msg=world').expect(200);
+    expect(res.body).to.containEql({greeting: 'Hello from LoopBack'});
+  });
```

Comment:
:robot: OpenAI

The test cases look good. No issues found.

<!-- This is an auto-generated comment by OpenAI -->